machineconfiguration/v1alpha1/0000_80_machineconfignode-TechPreviewNoUpgrade.crd.yaml

```diff
@@ -0,0 +1,203 @@
+              type: string
+            kind:
+              description: 'Kind is a string value representing the REST resource this object represents. Servers may infer this from the endpoint the client submits requests to. Cannot be updated. In CamelCase. More info: https://git.k8s.io/community/contributors/devel/sig-architecture/api-conventions.md#types-kinds'
+              type: string
+            metadata:
+              type: object
+            spec:
+              description: spec describes the configuration of this specific machineconfignode
+              type: object
+              properties:
+                nodeRef:
+                  description: nodeRef contains references to the node for this MachineConfigNode
+                  type: object
+                  required:
+                    - apiVersion
+                    - kind
+                    - name
+                  properties:
+                    apiVersion:
+                      description: apiVersion details the group and version of this resource.
+                      type: string
+                    kind:
+                      description: kind describes the type of object thsi reference points to
+                      type: string
+                    name:
+                      description: name describes what this object is called
+                      type: string
+                pool:
+                  description: pool is the MachineConfigPool this MachineConfigNode is describing
+                  type: object
+                  required:
+                    - apiVersion
+                    - kind
+                    - name
+                  properties:
+                    apiVersion:
+                      description: apiVersion details the group and version of this resource.
+                      type: string
+                    kind:
+                      description: kind describes the type of object thsi reference points to
+                      type: string
+                    name:
+                      description: name describes what this object is called
+                      type: string
+            status:
+              description: status describes the last observed state of this machineconfignode
+              type: object
+              required:
+                - configVersion
+              properties:
+                conditions:
+                  description: 'conditions represent the observations of a MachineConfigState''s current state. Known .status.conditions.type are: "UpdatePreparing", "UpdateInProgress", "UpdatePostAction", "UpdateCompleting", "Resuming", "Updated", "Ready", "Errored"'
```

Comment:
Hmm, this description doesn't seem to be updated based on the actual object below